scripts/kind-e2e/e2e.sh

```diff
@@ -111,9 +115,13 @@ function setup_broker() {
         helm --kube-context cluster1 install submariner-latest/submariner-k8s-broker --name ${SUBMARINER_BROKER_NS} --namespace ${SUBMARINER_BROKER_NS}
     fi
 
+    # FIXME: Shouldn't this be a dynamic namespace, not hard-coded "default"?
```

Comment:
This actually isn't related to this change..

Reply:
Okay, I removed it in the latest version.

Reply:
Long-ago done